ansible/playbooks/de-deploy-services.yaml

```diff
@@ -376,7 +376,7 @@
       msg: ":heavy_check_mark: Done deploying infosquito"
 
 - name: Redeploy job-status-to-apps-adapter
-  hosts: jexevents
+  hosts: job-status-to-apps-adapter:&systems
```

Comment:
It looks like some of @ianmcorvidae's commits became part of this PR somehow...
Better rebase and force-push this branch to fix.